python/ray/serve/controller.py

```diff
@@ -747,6 +750,11 @@ async def create_backend(self, backend_tag, backend_config,
                              replica_config):
         """Register a new backend under the specified tag."""
         async with self.write_lock:
+
+            # Ensures this method is idempotent.
+            if backend_tag in self.backends:
```

Comment:
We should probably also check that the configs are equal

Reply:
Sounds good. If the configs are not equal, should we raise ValueError again just like on the client side in `api.py`?  Or should we only raise that error on the client side?